config/sync/migrate_plus.migration.d7_file_media_image.yml

```diff
@@ -19,26 +19,33 @@ source:
   constants:
     bundle: image
 process:
-  mid: fid
   bundle: constants/bundle
   langcode:
     plugin: default_value
     source: language
     default_value: und
-  name: field_file_image_title_text/0/value
+  name: filename
   uid:
     -
-      plugin: skip_on_empty
-      method: process
-      source: uid
+      plugin: migration_lookup
+      migration: d7_file
+      source: uuid
```

Comment:
This works because we can lookup the file by the D7 uuid. We use the service when we need to find a D9 entity id given a D7 file id... going around the houses with the UUID values in the migrate map tables.